src/game/savegame.c

```diff
@@ -382,7 +389,7 @@ bool Savegame_Save(int32_t slot_num, GAME_INFO *game_info)
 
     for (int i = 0; i < g_GameFlow.level_count; i++) {
         if (g_GameFlow.levels[i].level_type == GFL_CURRENT) {
-            game_info->start[i] = game_info->start[g_CurrentLevel];
+            game_info->current[i] = game_info->current[g_CurrentLevel];
```

Comment:
Fixing this looks like it fixed the new save format not loading / restarting Lara's items / stats properly.